src/utils.ts

```diff
@@ -7,3 +7,41 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPages(arrItems: string[][]): number[] {
+  const result: number[] = [];
+  let count: number = 0;
```

Comment:
The `count` variable is redundant. You can directly use the loop index `i` in `result.push(i)`.